src/node.js

```diff
@@ -623,9 +623,7 @@
         stdout.emit('error', er);
       };
       if (stdout.isTTY) {
-        process.on('SIGWINCH', function() {
-          stdout._refreshSize();
-        });
+        process.on('SIGWINCH', stdout._refreshSize);
```

Comment:
Isn't `this` referring to the stream though? i.e. the `stdout` or `stderr` objects?